development/includes/_top-nav.pug

```diff
@@ -0,0 +1,15 @@
+nav(class="navbar navbar-default navbar-fixed-top")
+  div.container-fluid
+    div.navbar-header
+      button.navbar-toggle(type="button" data-toggle="collapse" data-target="#myNavbar")
+        span.icon-bar
+        span.icon-bar
+        span.icon-bar
+      a.navbar-brand(href="/index.html") MSPUDDING
+
+    div(class="collapse navbar-collapse" id="myNavbar")
```

Comment:
You should remove this surrounding div as it has no real utility. You can add the **id** to the `<ul>` tag below.